viewer.py

```diff
@@ -654,29 +767,50 @@ def __exit__(self, *args):
         self.signal.connect(self.slot)
 
 
-
 if __name__ == '__main__':
 
     app = pg.mkQApp()
 
     v = AtlasViewer()
     v.show()
 
-    path = os.path.dirname(__file__)
+    path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
For some reason I had issues creating .ma files without the .realpath call. I would get an empty string without it.